docs/source/commandline/commandline-all.rst

```diff
@@ -314,13 +594,50 @@ Example:
     verified; however, you can limit this list by specifying a specific driver
     name or *pattern* after the **-verifysoftware** command.
 
+    Example:
+        .. code-block:: bash
+
+            mame64 coco3 -verifysoftware
+            romset coco_cart:7cardstd is good
+            coco_cart:amazing: a mazing world of malcom mortar (1987)(26-3160)(zct systems).rom (16384 bytes) - NEEDS REDUMP
+            romset coco_cart:amazing is best available
+            coco_cart:amazing1: a mazing world of malcom mortar (1987)(26-3160)(zct systems)[a].rom (16384 bytes) - NEEDS REDUMP
+            romset coco_cart:amazing1 is best available
+            romset coco_cart:androne is good
+            ...
+
 .. _mame-commandline-getsoftlist:
 
 **-getsoftlist** / **-glist** [<*pattern*>]
 
     Posts to screen a specific software list which matches with the system name
     provided.
```

Comment:
The pattern here is matched against software list names, not system names.

Reply:
Okay, adjusted. I'll have this commit ready as soon as I deal with the rest of these.